.coveragerc

```
@@ -0,0 +1,11 @@
+[run]
+branch = True
+parallel = True
+source =
+    proxy_server
+    tests/proxy_server
```

```suggestion
    tests/unit
```

In lms we've [moved the unit tests into `tests/unit/`](https://github.com/hypothesis/lms/tree/master/tests/unit) (so `tests/unit/proxy_server/...`) so files like `.coveragerc` can just say `tests/unit` which means they're the same for every project

---

I noticed in the other repo it looks like the project files moved into a src directory? Would you like me to do that here too?

---

@jon-betts Can you remember what we decided about `src` directories? We're using them for the libs but not yet for the apps. I can't remember whether we decided we wanted to move the apps into `src` dirs

---

I think it makes less sense in the case of an app for sure. The argument for it is just that our scripting and boilerplate might be more similar. Then again, I think the divide between the apps and libs is likely to be large enough that we'll be maintaining two sets regardless.

---

Ok, so @hmstepanek no `src` dir needed